lib/screen/customer_support/support_thread_page.dart

```diff
@@ -605,12 +771,21 @@ class _SupportThreadPageState extends State<SupportThreadPage> {
     return result;
   }
 
+  getRating(String text) {
+    if (text == "${STAR_RATING}1") return 1;
```

Comment:
just a suggestion, maybe you could research using Regex, it might reduce the effort to use too many if here.

Reply:
this case is kinda simple and not to long to write or read so i think it's okay not to use Regex.

Reply:
it looks unprofessional here. imagine we have a 5-star rating, so it will go through 5 if commands. With clean code design, we aim to reduce the if..else.

I'm still fine if using switch case or check prefix then parse the number, but this looks weird.

Reply:
ok, i will change it to check prefix